### ISSUE TITLE
Italy: update admin 1 and add admin 2

### ISSUE THREAD

**Issue by KacperMadej** [highcharts/map-collection-dist#49]
A good information source is required.
A map source will also be appreciated.

Wikipedia (both English and Italian versions) pages about Regions of Italy are having some info and old maps, so those are not enough to be sure how should the new maps look.

Postponed until more data will be gathered.

**Comment by Ulrich0404** [comment]
We have some GeoJson that we are using for another project. It breaks down as Regions(sdmin1) with the proper counties (admin2) under those. As an example this would be the Region of Sardinia with it's underlying counties.
[Sardegna.zip](https://github.com/highcharts/map-collection-dist/files/4036913/Sardegna.zip)

We can provide the other 19 Regions if this format works.

Thank you.

**Comment by KacperMadej** [comment]
What we need for our map generator is a source is QGIS format with copyrights that will allow us to publish the maps based on those sources as a part of the free map collection.
In the past, we have used sources like Natura Earth.
We could also work with a map showing the changes and detailed description of new and updated region shapes and ISO codes.
The best option would be to have a nice QGIS source, but the work could be done manually if enough official or checked info will be given.

There are some maps in here: https://www.istat.it/it/archivio/222527
However, I am unable to find the maps that will be officially confirmed for usage from 2020.01.01.
Other than that the files are looking useful and usable. I will try to confirm the license for the maps there.

**Comment by Ulrich0404** [comment]
The OSM maps files I gave you earlier are definitely open source.  Here is where they came from 
https://wambachers-osm.website/boundaries/

What we liked about this source is that it has the nesting/marrying of Admin 1 to Admin 2 (Italian regions to counties) already done.

The ISTAT shape files are also updated with the new counties.  I can send you the new versions of the files we have used in the past.  However, those shape files have Admin 1 and Admin 2 separately.  Someone would have to marry the information.  We could document the relationships between the counties and region files.  The file we sent earlier was just an example of the marriage of Sardinia (region) with its new counties.  Please let me know if you want all the Italian regions in one Map, all  Italian counties in another map and each 20 regions with their counties.
To summarize: what we are seeing is that you currently have Admin 2 (Counties/Provinces) for Italy available, but it is out of date.  And you do not have Admin 1 (Regions) data at all.  So we would need you to update the Admin 2 and add Admin level 1, so we can use hierarchical maps.  This would also entail being able to see each region separately with it's own counties.  

Would it be possible for you to use one of the above sources -- including QGIS conversion, if that's needed on your end? 

Thanks.
Rose

**Comment by Ulrich0404** [comment]
Hi, 
What is your feedback on the subject? Please let us now.
Best, 
Rose

**Comment by KacperMadej** [comment]
@Ulrich0404 
We are in the middle of researching for a new and more up-to-date source of the maps. Open Street Map seems to be the best candidate currently, so we will be preparing a POC.

We could use any map for the tests, but I will try to set Italy as the test target, so when the maps will be created then I should be able to post them in this thread, so you don't have to wait for the next map collection release. There is not much more that I could do to speed up the map development process at this moment.

Please let me know which region (admin1) with its provinces (admin2) would you like to have access to the most in the first place for a map check or running any tests. Based on this info I will prioritize that region.

**Comment by Ulrich0404** [comment]
Hi, your approach is correct and we agree that OpenStreetMap is a very good resource. As mentioned we use this https://wambachers-osm.website/boundaries/ webiste. 
Now  for the POC we suggest to start with a very stable Region Veneto with its Provinces: Venezia (VE), Belluno (BL), Padova (PD), Rovigo (RO), Treviso (TV), Verona (VR), Vicenza (VI).  Once succesfully completed this Region quickly enough you can cover all the other Regions highlighted in light blue.  Sardegna follows the same administrative hierarchy but its Provinces have been redesigned in 2017 (Cagliari (CA), Nuoro (NU), Oristano (OR), Sassari (SS), Sud Sardegna (SU)) . The most complicated will be Friuli Venezia Giulia but for now let's start the POC with Veneto. Attached a screen shot. If we can be of any help please let us know.Thanks, Rose
![Italian Regions with Veneto Provinces](https://user-images.githubusercontent.com/49291568/72739603-ee163f00-3b71-11ea-974f-cca8dbc41082.png)
